includes/pages/user_myshifts.php

```diff
@@ -20,10 +20,19 @@ function user_myshifts()
 {
     $user = auth()->user();
     $request = request();
+    $is_angeltype_supporter = false;
+    if ($request->has('edit')) {
+        $id = $request->input('edit');
+        $shiftEntry = ShiftEntry::where('id', $id)
+            ->where('user_id', User::find($request->input('id'))->id)
+            ->with(['shift', 'user'])
+            ->first();
+        $is_angeltype_supporter = auth()->user()->isAngelTypeSupporter($shiftEntry->angelType);
```

Comment:
We should check here if the shift entry really exists:
```
Attempt to read property "angelType" on null
```

```suggestion
        $is_angeltype_supporter = $shiftEntry && auth()->user()->isAngelTypeSupporter($shiftEntry->angelType);
```